dd-sdk-android/src/main/json/trace/span-schema.json

```diff
@@ -0,0 +1,210 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "span-schema.json",
+  "title": "SpanEvent",
+  "type": "object",
+  "description": "Schema of a Span Event",
+  "required": [
+    "trace_id",
+    "span_id",
+    "parent_id",
+    "resource",
+    "name",
+    "service",
+    "duration",
+    "start",
+    "error",
+    "type",
+    "meta",
+    "metrics"
+  ],
+  "properties": {
+    "trace_id": {
+      "type": "string",
+      "description": "The id of the trace this Span belongs to",
+      "readOnly": true
+    },
+    "span_id": {
+      "type": "string",
+      "description": "The unique id of this Span",
+      "readOnly": true
+    },
+    "parent_id": {
+      "type": "string",
+      "description": "The id of the parent Span in this trace",
+      "readOnly": true
+    },
+    "resource": {
+      "type": "string",
+      "description": "The resource name"
+    },
+    "name": {
+      "type": "string",
+      "description": "The name of this Span"
+    },
+    "service": {
```

Comment:
this one should be read only as it should match the service name of all other features, logs, RUM etc…